test_conformance/events/test_event_dependencies.cpp

```diff
@@ -275,6 +276,11 @@ int test_event_enqueue_wait_for_events_run_test(
 
     for (loop_count = 0; loop_count < TEST_COUNT; loop_count++)
     {
+        int event_count = 0;
+        clEventWrapper first_dependency = (loop_count == 0) ? pre_loop_event : last_loop_event;
+        clEventWrapper event[SIZE]; // If we usemarkers we get 2 more events per iteration
```

Comment:
Don't think we need a new `SIZE` macro since it's only ever used here. Would just make this line:

```cpp
// A maximum of 5 events are created in the loop
clEventWrapper event[5];
```

Reply:
thank you done!